app/products/calls/components/call_notification/call_notification.tsx

```diff
@@ -135,6 +137,10 @@ export const CallNotification = ({
         }
     }, []);
 
+    useEffect(() => {
+        playIncomingCallsRinging(incomingCall.serverUrl, incomingCall.callID);
+    }, [incomingCall.serverUrl, incomingCall.callID, appState]);
```

Comment:
ok so help me here

1. if the app comes to the foreground, ring
2. if the app goes to the background, ring.
3. app is closed, not ring at all.

yes?

Reply:
1. correct
2. incorrect: no ringing ever in the background/app switcher (which is handled in `shouldRing`)
3. correct